packages/flutter/lib/src/widgets/automatic_keep_alive.dart

```diff
@@ -89,6 +89,10 @@ class _AutomaticKeepAliveState extends State<AutomaticKeepAlive> {
         // build of this subtree. Wait until the end of the frame to update
         // the child when the child is guaranteed to be present.
         SchedulerBinding.instance.addPostFrameCallback((Duration timeStamp) {
+          // The element is no longer alive
```

Comment:
End comments with a period. Also since the comment is outside the if it should be a question ("Is the element still alive?"), or you can put it inside the block, or you can just remove it, the code speaks for itself here and the comment isn't really adding anything.